addon/components/polaris-color-picker/slidable.js

```diff
@@ -4,6 +4,7 @@ import { computed } from '@ember/object';
 import { isNone, typeOf } from '@ember/utils';
 import { htmlSafe } from '@ember/string';
 import layout from '../../templates/components/polaris-color-picker/slidable';
+import { getRectForNode } from '@smile-io/ember-polaris/utils/geometry';
```

Comment:
I'd previously ignored the React `getRectForNode` util because in our usage up to this point it's only ever been calling `getBoundingClientRect` on the passed-in node, but now that `sticky-manager` passes the `document` element (which doesn't support `getBoundingClientRect`) I figured we should match what the React implementation does everywhere, to make future updates cleaner.